frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -47,6 +48,9 @@ export const fetchParcelsDetail = (params: API.IPropertySearchParams) => (dispat
   return CustomAxios()
     .get(ENVIRONMENT.apiUrl + API.PARCELS_DETAIL(params))
     .then((response: AxiosResponse) => {
+      if (response?.data !== undefined && response.data.length > 0) {
```

Comment:
store the parcel detail in redux so that multiple consumers can display the result (the original caller, as well as the map.

Reply:
The map doesn't use this type of model anymore.  It uses a FeatureCollection

Reply:
Yeah, I know I've been following the reviews. I can't exactly pre-emptively consume that without breaking everything in this branch though, its just going to be a fair amount of work to merge.